src/token.rs

```diff
@@ -153,6 +172,36 @@ fn make_token<P: Serialize + Deserialize + 'static>(subject: &str,
                              }))
 }
 
+/// Verify that the service requested for is allowed in the configuration
+fn verify_service(config: &Configuration, service: &str) -> Result<(), Error> {
+    if !config.audience.contains(&FromStr::from_str(service)?) {
```

Comment:
Yes. This is not `String::contains`. It's this: https://github.com/lawliet89/rowdy/pull/23/files#diff-8eaf0fafad7d2ed9ec9a701c04424c96R412